insalan/tournament/models/match.py

```diff
@@ -0,0 +1,113 @@
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+from django.contrib.postgres.fields import ArrayField
+from typing import List
+import math
+
+class MatchStatus(models.TextChoices):
+    """Information about the status of a match"""
+
+    SCHEDULED = "SCHEDULED", _("Prévu")
+    ONGOING = "ONGOING", _("En cours")
+    COMPLETED = "COMPLETED", _("Terminé")
+
+class BestofType(models.TextChoices):
+    """Best of type for a match"""
+
+    BO1 = "BO1", _("Série de 1")
+    BO3 = "BO3", _("Série de 3")
+    BO5 = "BO5", _("Série de 5")
+    BO7 = "BO7", _("Série de 7")
```

Comment:
Wouldn't it be possible to only have `BO` with an int field to have a more dynamic way of doing this, especially if you're already doing math which are pretty close for ranking?

Reply:
Yes, you are right it is way better to have int field.